src/StoreKeeper/WooCommerce/B2C/PaymentGateway/StoreKeeperBaseGateway.php

```diff
@@ -149,6 +160,21 @@ private function getPaymentUrl(\WC_Order $order)
         return $payment['payment_url'];
     }
 
+    protected static function getRedirectUrl($order_id): string
+    {
+        $url = add_query_arg(
+            [
+                'wc-api' => 'backoffice_pay_gateway_return',
+                'utm_nooverride' => '1',
+                'wc-order-id' => $order_id,
+            ],
+            home_url('/')
+        );
+        $url .= '&trx={{trx}}';
```

Comment:
Why is trx appended instead of used inside `add_query_arg`?

Reply:
append instead of add_query_arg, so it's not encoded into %7B%7Btrx%7D%7D